src/openforms/js/components/admin/Keyboard.js

```diff
@@ -0,0 +1,25 @@
+export const saveAndContinueButtonQuery = "form input[type=submit][name=_continue]";
+
+const saveAndContinueHandler = (event) => {
+    // because of the way React initializes we run the querySelector when the buttons is pressed
+    const button = document.querySelector(saveAndContinueButtonQuery);
+    if (!button) {
+        console.warn("Cannot find 'Save & Continue' button");
+    } else {
+        event.preventDefault();
+        button.click();
+    }
+}
+
+const enableKeyboardShortcuts = () => {
+    window.addEventListener('DOMContentLoaded', (e) => {
+        document.addEventListener('keydown', event => {
```

Comment:
I'm also wondering if we shouldn't tap into `keyup` rather than `keydown`? That triggers the save when the keys are released.